index.js

```diff
@@ -35,6 +44,89 @@ app.post('/api/item/reserve/:id', (req, res) => {
   res.send();
 });
 
+app.post('/api/register', async (req, res) =>{
+const {name, email, phone_number, password} = req.body;
+
+
+if (email == "" || name == "" || phone_number == "" || password == ""){
```

Comment:
I think we decided that phone numbers will be optional, or we're at least not going to require one upon registration (for now). Someone can correct me on this if I'm wrong though

Reply:
Fixed.